### PR TITLE
Link to FAQ to find test centers (EXPOSUREAPP-8086)

### PR DESCRIPTION
**Pull request by harambasicluka**
Implementation

image

image

_Note:_
The image on the side is not perfectly aligned as in the mockups but this would require a refactoring of this cards, this also affects the already implemented cards. To bring it a little bit closer to the mockups I changed the navigation colors.

### CODE REVIEW

**Comment by vaubaehn** [comment]
You prefer opening FAQ over directly jumping to the map? As a fallback reason in case map service is interrupted/terminated? But TSI could provide a message on their site then. Like this it takes one click more effort to get to the map.

But nice to implement it in general 👍

**Review by mtwalli — DISMISSED** [review]
Lgtm, tested on Android 6 emulator 😃

**Comment by harambasicluka** [comment]
> You prefer opening FAQ over directly jumping to the map? As a fallback reason in case map service is interrupted/terminated? But TSI could provide a message on their site then. Like this it takes one click more effort to get to the map.
> 
> But nice to implement it in general 👍

@vaubaehn the FAQ entry contains multiple links and we want to be able to adjust the links without releasing a new app version :)

**Comment by vaubaehn** [comment]
@harambasicluka 
Thanks, I simply forgot about the PCR tests, which are not necessarily included in the map. Also, the link to @Ein-Tim 's list is also worth to jump to FAQ :)

**Comment by harambasicluka** [comment]
> @harambasicluka
> Thanks, I simply forgot about the PCR tests, which are not necessarily included in the map. Also, the link to @Ein-Tim 's list is also worth to jump to FAQ :)
> 
> Edit: why are you laughing about Android 6 emulator? It's important!! 😉

It was more like an internal joke, because it's my first implementation in a while :)

File: Corona-Warn-App/src/main/res/values-de/antigen_strings.xml [hunk 1]
```diff
@@ -5,9 +5,9 @@
         Homescreen cards - register test card
     ###################################### -->
     <!-- XHED: Register test homescreen card: title -->
-    <string name="ag_homescreen_card_test_register_title">"Nächste Schritte"</string>
+    <string name="ag_homescreen_card_test_register_title">"Weiter"</string>
     <!-- XTXT: Register test homescreen card: body -->
-    <string name="ag_homescreen_card_test_register_body">"Nutzen Sie die App, um Ihr Testergebnis abzurufen und andere schneller warnen zu können, oder um Ihr COVID-Testzertifikat anzufordern."</string>
+    <string name="ag_homescreen_card_test_register_body">"Finden sie Teststellen in ihrer Umgebung und registrieren sie einen Test, um andere schneller warnen zu können, oder ihr digitales COVID-Testzertifikat anzufordern."</string>
```
Comment (DerVogel2020) [inline Corona-Warn-App/src/main/res/values-de/antigen_strings.xml:10]: Sie, Ihrer und Ihr müssten groß geschrieben werden.

**Review by DerVogel2020 — COMMENTED** [review]
This will fix https://github.com/corona-warn-app/cwa-wishlist/issues/498

**Comment by DerVogel2020** [comment]
Will this fix Internal Tracking ID: EXPOSUREAPP-7204?

File: Corona-Warn-App/src/main/res/values-de/antigen_strings.xml (hunk 1 above)
Comment (NataliaLemmerth) [inline Corona-Warn-App/src/main/res/values-de/antigen_strings.xml:10]:
```suggestion
    <string name="ag_homescreen_card_test_register_body">"Finden Sie Teststellen in Ihrer Umgebung und registrieren Sie einen Test, um andere schneller warnen zu können, oder Ihr digitales COVID-Testzertifikat anzufordern."</string>
```

**Review by NataliaLemmerth — CHANGES_REQUESTED** [review]
Just these small text changes. Everything else is fine.

**Review by BMItr — DISMISSED** [review]
nothing to complain 👌

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Comment by harambasicluka** [comment]
> Will this fix Internal Tracking ID: EXPOSUREAPP-7204?

@DerVogel2020 Yes! You can already check the URL: https://www.coronawarn.app/de/faq/#where_can_i_get_tested

**Pull request merged**